src/app/common/services/configuration-service.coffee

```diff
@@ -0,0 +1,14 @@
+angular.module("doubtfire.common.services.configuration", [])
+
+.constant('external_name', '/* @echo EXTERNAL_NAME */')
+#
+# Services for a configurable external name
+#
+.factory("configurationService", (external_name) ->
```

Comment:
No need for a service here 😊 We could just inject `externalName` anywhere we need to, like how we do with the `apiUrl`.